Samples/WinMLSamplesGallery/WinMLSamplesGalleryNative/Win32Application.cpp

```diff
@@ -0,0 +1,166 @@
+#include "pch.h"
+#include "stdafx.h"
+#include "Win32Application.h"
+
+HWND Win32Application::m_hwnd = nullptr;
+bool Win32Application::close_window = false;
+
+// Get the HModule that will be used to spawn
+// the HWND
+HMODULE GetCurrentModule()
+{
+    HMODULE hModule = NULL;
+    GetModuleHandleEx(
+        GET_MODULE_HANDLE_EX_FLAG_FROM_ADDRESS,
+        (LPCTSTR)GetCurrentModule,
+        &hModule);
+    return hModule;
+}
+
+// Launch the HWND and init D3D
+int Win32Application::Run(D3D12Quad* pSample, int nCmdShow)
```

Comment:
Hungarian warts like `p` were already deprecated as far back as 2007 in SDK samples. Can we kill them for good in 2022?